Chat/ClientApp/src/core/sideEffects.ts

```diff
@@ -58,12 +58,25 @@ const addUserToThread = (displayName: string, emoji: string) => async (dispatch:
   // create our user
   let userToken = await getToken();
 
+  var test = await refreshTokenAsync(userToken.identity);
+
   if (userToken === undefined) {
     console.error('unable to get a token');
     return;
   }
 
-  let userAccessTokenCredentialNew = new AzureCommunicationUserCredential(userToken.token);
+ let options: RefreshOptions = {
+  initialToken: userToken.token,
+  tokenRefresher: async () : Promise<string> => { 
```

Comment:
Can this be simplified?
```suggestion
  tokenRefresher: () => refreshTokenAsync(userToken.identity)
```

That should do the exact same as your code.